composer.json

```diff
@@ -13,9 +13,12 @@
         "keboola/db-extractor-common": "^9.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "7.*",
         "ext-pdo_pgsql": "*",
-        "aws/aws-sdk-php": "^3.18"
+        "aws/aws-sdk-php": "^3.18",
+        "jakub-onderka/php-parallel-lint": "^1.0",
+        "keboola/coding-standard": "^8.0",
+        "phpstan/phpstan-shim": "^0.11.4",
+        "phpunit/phpunit": "7.*"
```

Comment:
Mě nějaký důvod, že tohle je jiný formát? (vs. `^7.0`)

Reply:
Asi nema, takhle to tam bolo